hummingbot/connector/derivative/dydx_perpetual/dydx_perpetual_client_wrapper.py

```diff
@@ -93,6 +95,15 @@ async def get_account(self):
                                                      ethereum_address=self._ethereum_address))
         return await f
 
+    async def get_funding_payments(self, market: str, before_ts: float):
+        iso_ts = datetime.datetime.utcfromtimestamp(before_ts).isoformat()
+        f = self._loop.run_in_executor(None, partial(self.client.private.get_funding_payments,
+                                                     market=market,
+                                                     limit=100,
+                                                     effective_before_or_at=iso_ts
+                                                     ))
+        return await f
```

Comment:
The use of `partial` does not seem to be required here. The `run_in_executor` function already allows to specify a function to be run and the arguments for that function. In the following link there is an example: https://github.com/calebmadrigal/asyncio-examples/blob/master/run_in_executor.py

Reply:
Looking deeper into `run_in_executor` it appears that python docs suggest that we use `partial` if we want to pass kwargs to the function.
https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.run_in_executor